a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -93,14 +93,17 @@ def get_create_load_balancer_flow(self, topology, listeners=None):
         lb_create_flow.add(
             self.get_post_lb_vthunder_association_flow(
                 post_amp_prefix, topology, mark_active=(not listeners)))
-
-        lb_create_flow.add(handler_virtual_server.CreateVitualServerTask(
+        lb_create_flow.add(a10_database_tasks.MarkVThunderStatusInDB(
```

Comment:
for naming consistency - you can either follow -> `lb_create_flow` or `create_lb_flow`. Similary, `lb_delete_flow` or keep it `delete_lb_flow`